earth_enterprise/src/fusion/autoingest/AssetFactory.h

```diff
@@ -256,6 +257,199 @@ namespace AssetFactory
   }
 
 
+  template <class MutableDerivedVersionHandleType, class Version,
+            class ConfigType, class VersionDType, class Extras = int>
```

Comment:
The `Extras` argument is not used. We should take it out.

Reply:
I think the `Version` argument is also unused. That should be taken out of all the functions as well